app/javascript/components/widgets/messages/message.jsx

```diff
@@ -5,9 +5,14 @@ import { GreySubText } from '@components/typography';
 import { fontSizes, leftPanelWidth } from '@lib/theme';
 import sunMoon from '@images/sunmoon.png';
 
-export const LoadingMessage = () => <Message message="Loading..." />;
+export const LoadingMessage = () => (
+  <Message message="Loading..." height={(window.innerHeight - 100) / 2} />
```

Comment:
Although this likely works, I suspect its solving the symptom rather than the disease. I'd rather resolve the css issue that causes are containing width to be wrong than add an inline style